webauth/agent.js

```diff
@@ -17,16 +17,21 @@ export default class Agent {
         resolve(event.url);
       };
       Linking.addEventListener('url', urlHandler);
-      NativeModules.A0Auth0.showUrl(url, closeOnLoad, (error, redirectURL) => {
-        Linking.removeEventListener('url', urlHandler);
-        if (error) {
-          reject(error);
-        } else if(redirectURL) {
-          resolve(redirectURL);
-        } else if(closeOnLoad) {
-          resolve();
+      NativeModules.A0Auth0.showUrl(
+        url,
+        closeOnLoad,
+        disableAuthenticatedSession,
+        (error, redirectURL) => {
+          Linking.removeEventListener('url', urlHandler);
+          if (error) {
+            reject(error);
+          } else if (redirectURL) {
+            resolve(redirectURL);
```

Comment:
what about joining the last 2 cases like this?

```
else if (redirectURL || closeOnLoad) {
  resolve(redirectURL);
}
```

Reply:
This code is untouched in this PR, so I'll leave it outside of the scope of this PR